Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -0,0 +1,62 @@
+{ Expression } = require './expression'
+{ FunctionRef } = require './reusable'
+{ ValueSet, Code } = require '../datatypes/datatypes'
+{ build } = require './builder'
+
+module.exports.Quantity = class Quantity extends Expression
+  constructor: (json) ->
+    super
+    @unit = json.unit
+    @value = json.value
+
+  exec: (ctx) ->
+    @  
+  sameOrBefore: (other) ->
```

Comment:
I understand why you did this (for consistency w/ DateTime API), but it really feels awkward.  I'd almost rather see `lessThan`, `lessThanOrEqual`, etc, and change `DateTime` to be consistent with that.  If/when we support numbers better (using `BigDecimal` or something like that), we'll need to override these operators there too.

I won't hold up the PR on account of this, but it's something to consider for later.